lib/matplotlib/cbook/__init__.py

```diff
@@ -280,6 +286,31 @@ def process(self, s, *args, **kwargs):
                     else:
                         raise
 
+    @contextlib.contextmanager
+    def block(self, *, signal=None):
+        """
+        Block callback signals from being processed.
+
+        A context manager to temporarily block/disable callback signals
+        from being processed by the registered listeners.
+
+        Parameters
+        ----------
+        signal : str, default: None
+            The callback signal to block, default blocks all signals
```

Comment:
```suggestion
        signal : str, default: None
            The callback signal to block, default is to block all signals
```
yes correcting myself here, but I'm not sure which is the correct grammatical construction.

Reply:
I put it back to optional, which is apparently preferred if the value isn't used explicitly as a value I think...? https://numpydoc.readthedocs.io/en/latest/format.html#parameters

I also made it into two sentences which may be easier to follow now. No worries on the nits if you have other suggestions. Better to get it right the first time.